network/httpd_proxy.py

```diff
@@ -0,0 +1,344 @@
+    required: false
+    default: /balancer-manager/
+  disabled:
+    description:
+      - Immediately terminates all the sessions attached to the specified balancer member, making it unreachable.
+        (Requires member_set_status: true).
+    required: false
+    default: false
+  draining_mode:
+    description:
+      - Prevents new sessions from being attached the the specified balancer member.
+        (Requires member_set_status: true).
+    required: false
+    default: false
+  enabled:
+    description:
+      - Enable the balancer member with all attributes amongst [draining_mode,ignore_errors,disabled,hot_standby] set to false.
+        Mutually exclusive with [disabled,draining_mode,hot_standby,ignore_errors].
+        (Requires member_set_status: true).
+    required: false
+    default: false
+  hot_standby:
+    description:
+      - Puts the balancer member in a temporary disabled state.
+        This member enabled state will only trigger when the balancer pool has no other available enabled members.
+        (Requires member_set_status: true).
+    required: false
+    default: false
+  ignore_errors:
+    description:
+      - Ignore all HTTP replies errors from the member, preventing automatic disabling of this member from the balancer pool.
+        (Requires member_set_status: true).
+    required: false
+    default: false
+  member_host:
+    description:
+      - [ipv4|ipv6|fqdn] of the balancer member to get or to set attributes to.
+        Port number is autodetected and should not be specified here.
+        If undefined, httpd_proxy module will return a "members" list of dictionaries of all the current balancer pool members' attributes.
+    required: false
+    default: None
+  member_set_status:
+    description:
+      - Flag needed to set a balancer pool member status.
+        (Requires member_host).
+    required: false
+    default: false
+'''
+
+EXAMPLES = '''
+# Get all current balancer pool members' attributes:
+- httpd_proxy: balancer_vhost=10.0.0.2
+
+# Get a specific member's attributes:
+- httpd_proxy: balancer_vhost=myws.mydomain.org balancer_suffix="/lb/" member_host=node1.myws.mydomain.org
+
+# Enable all balancer pool members:
+- httpd_proxy: balancer_vhost="{{ myloadbalancer_ip }}"
+  register: result
+- httpd_proxy: balancer_vhost="{{ myloadbalancer_ip }}" member_host="{{ item.ip }}" member_set_status=true enabled=true
+  with_items: "{{ result.members }}"
+
+# Gracefully disable a member from a loadbalancer node:
+- httpd_proxy: balancer_vhost="{{ vhost_ip }}" member_host="{{ member.ip }}" member_set_status=true draining_mode=true delegate_to=myloadbalancernode
+- wait_for: host="{{ member.ip }}" port={{ member.port}} state=drained delegate_to=myloadbalancernode
+- httpd_proxy: balancer_vhost="{{ vhost_ip }}" member_host="{{ member.ip }}" member_set_status=true disabled=true delegate_to=myloadbalancernode
+
```

Comment:
This next blurb could be reformatted and used for the 'RETURN' documentation, per http://docs.ansible.com/ansible/developing_modules.html#module-documenting